InstallGuide.md

```diff
@@ -1,5 +1,7 @@
-# Up to date installation directions and manifests can be found in the [Calico docs](https://docs.projectcalico.org/v2.6/getting-started/kubernetes/installation/hosted/canal)
+# [Canal manifests and docs](https://docs.projectcalico.org/v2.6/getting-started/kubernetes/installation/hosted/canal) have moved to [docs.projectcalico.org](https://docs.projectcalico.org/)
 
+Up to date installation directions and manifests can be found
+[here](https://docs.projectcalico.org/v2.6/getting-started/kubernetes/installation/hosted/canal).
```

Comment:
[according to the style guide](https://github.com/projectcalico/calico/blob/master/DOC_STYLE_GUIDE.md#link-text) we shouldn't use `here` as the link text